x-pack/plugins/fleet/server/services/files/mocks.ts

```diff
@@ -86,7 +86,7 @@ export const createFromHostEsSearchResponseMock =
         max_score: 0,
         hits: [
           {
-            _index: '.fleet-fileds-fromhost-meta-foo-000001',
+            _index: '.fleet-fields-fromhost-meta-foo-000001',
```

Comment:
Not related to this change but I found this typo that I had to fix.

Reply:
Actually it is not a typo: https://github.com/elastic/kibana/blob/000b70c000ec1c8ce9d2ccda6faf60247116a263/x-pack/plugins/fleet/common/constants/file_storage.ts#L11

Reply:
Let's pretend it means "Fleet File Datastream" 🤫

Reply:
Oh! I'll leave it as it is then. 😅